### PR TITLE
upgrade helm to v3.9.4

### PR DESCRIPTION
**Pull request by justadogistaken**
upgrade the helm to v3.9.4 to fix the waring in the repository.

@kakaZhou719 this changes have impact on "debug", do you know how to test it?

### CODE REVIEW

**Comment by kakaZhou719** [comment]
@justadogistaken ,this changes have impact on "sealer debug", just run the cli cmd to test it.  meanwhile, it is not a sealer core feature.

File: apply/driver/local.go [hunk 1]
```diff
@@ -159,7 +159,7 @@ func (applier *Applier) reconcileCluster() error {
 	}
 	defer func() {
 		if err := applier.unMountClusterImage(); err != nil {
-			logrus.Warnf("failed to umount image(%s): %v", applier.ClusterDesired.ClusterName, err)
+			logrus.Warnf("failed to umount cluster(%s): %v", applier.ClusterDesired.Name, err)
```
Comment (justadogistaken) [inline apply/driver/local.go:162]: @kakaZhou719 @allencloud Help to consider if the ClusterName is needed. In new k8s/api version, the ClusterName is removed. IMO, we have the Cluster struct already, no need to have the ClusterName filed.

Comment (kakaZhou719) [reply apply/driver/local.go:162]: agree with you. `cluster.Name` is meaningful to us rather than `cluster.ClusterName`

Comment (allencloud) [reply apply/driver/local.go:162]: +1

**Comment by allencloud** [comment]
@justadogistaken conflict happens, please rebase and re-push it. Thanks.

File: apply/driver/local.go (hunk 1 above)
Comment (justadogistaken) [reply apply/driver/local.go:162]: I checked this. This is a weird logic.
The ClusterName is just used in this place.
I guess someone misunderstand the ClusterName and Name.

**Review by kakaZhou719 — APPROVED** [review]

**Pull request merged**